### ISSUE TITLE
Proposal: update and harmonize global nav

### ISSUE THREAD

**Issue by lucasgonze** [magma/magma#13386]
## Problem

The top nav in docusaurus isn't representative of the current information architecture of Magma documentation. It should be updated. The structure should be shared with top-level navigation in the contributor's guide.

The content of the top nav is:
- Home (https://magmacore.org/)
- Docs (https://docs.magmacore.org/)
- Code (https://github.com/magma)
- Community (https://magmacore.org/join-the-open-source-community)

<img width="410" alt="image" src="https://user-images.githubusercontent.com/111581/180331196-445294fd-0b20-4354-a530-b36176195519.png">

Issues to address:
- Given that this document is https://docs.magmacore.org/, the "home" link should probably be https://docs.magmacore.org/
- The link to https://magmacore.org/ needs a more descriptive title, like "About"
- The convention for linking to Github projects might to title them "Github" instead of "code"
- The Community link doesn't add much, since it reiterates the magmacore.org link.
- There should be a link to the contributor guide. 
- If possible there should be a shared nav for all subsites
- There is no path to the wiki hosted by Linux Foundation

## Solution

I propose this new nav bar:
- About (https://magmacore.org/): top-level landing page for the project as a whole (also called "Magma website")
- Using (https://docs.magmacore.org/): deployment and usage help (also called "Docusaurus")
- Contributing (https://github.com/magma/magma/wiki/): : contributor guide
- Code (https://github.com/magma): source control
- Workstreams (https://wiki.magmacore.org/): meeting notes and project management (also called "Confluence")

The descriptions can be link titles visible on mouseover.

This nav bar can be the custom footer in the Contributors Guide:
<img width="834" alt="image" src="https://user-images.githubusercontent.com/111581/180332437-1e598589-a0a0-4f6d-a16b-12fdf822fc60.png">

It can also be the content of the landing page in the Contributors Guide. See 
https://github.com/magma/magma/issues/13407

## Non-Goals

If the purpose includes a unified information architecture across subsites, it would make sense to integrate https://wiki.magmacore.org with magmacore.org, docs.magmacore.org/, and github.com/magma/magma/wiki/. However adding another subsite into this project may be an obstacle to completion. There is a lot of information on wiki.magmacore.org. Whether and how it should be integrated with other subsites should be left for the future.

There is a related issue: [User confusion between the two wikis #13409](https://github.com/magma/magma/issues/13409)

**Comment by Neudrino** [comment]
- The keyword `About` is already used on the website for a different target. This would introduce an inconsistency.
- Imho generic names should be preferred over tool names. E.g. Source code rather than Github; Documentation rather than DocuSaurus.
  - This is easier to understand for people, which do not know the specific tooling.
  - This allows for more stable link names in case of tooling changes.
- I find nouns mostly more descriptive and precise than verbs.
- Imho the official website should be used as master for all naming. We have the following names there for some of the above mentioned categories:
  - View the documentation
  - Get the source code

**Comment by ildikov** [comment]
My 2 cents:

- 'Home' on the docs site is indeed not the most intuitive, but I agree with @Neudrino that 'About' is not a good replacement. I think leaving it as 'Home' is probable fine, we could rename it to 'Magma Website' but it might be long.
- I would leave 'Docs' and 'Code' as is, I think they serve the purpose.
- I like 'Contributing (https://github.com/magma/magma/wiki/)'
- I would refer to Confluence as 'Wiki'. 'Workstreams' doesn't really describe where someone will get and what to expect there. I'm also not sure that this term is currently widely used within the community.

In summary:
Home or Magma Website (https://magmacore.org/): Magma website
Docs (https://docs.magmacore.org/): Deployment, configuration and usage information
Code (https://github.com/magma): Source code
Contributing (https://github.com/magma/magma/wiki/): Contributor Guide
Wiki (https://wiki.magmacore.org/): Meeting notes and project team resources

**Comment by maxhbr** [comment]
That all sounds reasonable. Maybe just do it and I do not think that it needs a TSC decision if it is sufficiently reviewed and discussed.

**Comment by lucasgonze** [comment]
Nice teamwork.

A task list to complete this in each of the subsites:
- [ ] Magma website
- [ ] Docs
- [ ] Code
- [ ] Contributing
- [ ] Wiki

**Comment by lucasgonze** [comment]
Modifying the Magma website to reflect this unified information architecture is a can of worms. Let's leave that for the future.

**Comment by lucasgonze** [comment]
As far as updating the info arch of our Github, the place is the /magma/magma README. 

The existing "Contributing" section was already in need of an update, since all the links pointed to stale content.

**Comment by lucasgonze** [comment]
I propose we change the title of https://github.com/magma/magma/wiki/ to be in sync with this scheme. Before this scheme the page title is "Magma project overview." Since the new name for this resource in the global nav is "Contributing", the page title should also be "Contributing."

**Comment by ildikov** [comment]
> The Confluence (aka Wiki) doesn't really have a place for a project-wide information architecture.

There should be an opportunity to personalize the Confluence landing page to the community's needs and preference. Currently the https://wiki.magmacore.org/ link defaults to the 'All Updates' tab, which hides all the content of the wiki. It's not intuitive and makes it hard to navigate through the wiki.

An example that comes to my mind is the wiki of the Anuket project: https://wiki.anuket.io

I'm sure that the LF staff can help if the community can agree on how the landing page on Confluence should look like.

**Comment by ildikov** [comment]
> I propose we change the title of https://github.com/magma/magma/wiki/ to be in sync with this scheme. Before this scheme the page title is "Magma project overview." Since the new name for this resource in the global nav is "Contributing", the page title should also be "Contributing."
> 
> We could use the content of the"Contributor Guide" article as the content of the home page at magma/magma/wiki/ . Basically the magma/wiki/ subsite would become a synonym for the contributor guide, which is in sync with the overall intention in this ticket.
> 
> @ildikov Thoughts?

I'm honestly good either way.

Since we can't change the name of the tab, which will remain 'Wiki', it kind of makes sense to list useful resources on the opening page for those who found Magma through the GitHub repo.

If we decide to rather have that page to be the 'Intro' section of the Contributor Guide, that can work too. In that case we should include the links on that page so people can find other project resources as well.

**Comment by lucasgonze** [comment]
> In that case we should include the links

I'll look at a combined page with both the links and the text.

**Comment by lucasgonze** [comment]
Fixed
